extension/src/popup/views/__tests__/MnemonicPhrase.test.tsx

```diff
@@ -103,9 +112,7 @@ describe("MnemonicPhrase", () => {
           screen.getByTestId("display-mnemonic-phrase-confirm-btn"),
         );
       });
-      await waitFor(() =>
-        screen.getByText(`redirect ${ROUTES.mnemonicPhraseConfirmed}`),
-      );
+      await waitFor(() => screen.getByText(`redirect ${ROUTES.pinExtension}`));
```

Comment:
after confirming mnemonic on a new account we're going straight to the pinExtension view